src/iop/demosaic.c

```diff
@@ -5767,12 +5767,10 @@ void gui_init(struct dt_iop_module_t *self)
 
   GtkWidget *box_raw = self->widget = gtk_box_new(GTK_ORIENTATION_VERTICAL, DT_BAUHAUS_SPACE);
 
-  g->demosaic_method_bayer = dt_bauhaus_combobox_from_params(self, "demosaicing_method");
-  for(int i=0;i<7;i++) dt_bauhaus_combobox_remove_at(g->demosaic_method_bayer, 9);
+  g->demosaic_method_bayer = dt_bauhaus_combobox_from_params_enum_interval(self, "demosaicing_method", 0, 9);
```

Comment:
I think the correct values are 0, 8
and below 9 15.

Also would be better to be able to use DT_IOP_DEMOSAIC_PPG, DT_IOP_DEMOSAIC_PASSTHROUGH_COLOR
and DT_IOP_DEMOSAIC_VNG, DT_IOP_DEMOSAIC_PASSTHR_COLORX.

But be careful those are not continuous numbers.

Reply:
Those numbers are indices, that index those declarations, they do not correspond to values of those enums so they are contiguous. Also end 9 is correct because i got inspired by c++ iterators and end is not included into selection. You can see that in the loop there is `i < end` therefore 9 is not included